.circleci/config.yml

```diff
@@ -3,46 +3,21 @@ version: 2.1
 orbs:
   hmpps: ministryofjustice/hmpps@1.1.3
   snyk: snyk/snyk@0.0.12
+  node: circleci/node@4.1.0
 
 jobs:
   build:
     executor: hmpps/node
     steps:
       - checkout
-      - run:
-          name: Update npm
-          command: 'sudo npm install -g npm@latest'
-      - restore_cache:
-          key: dependency-cache-{{ checksum "package-lock.json" }}
-      - run:
-          name: Install Dependencies
-          command: npm ci
-      - save_cache:
-          key: dependency-cache-{{ checksum "package-lock.json" }}
-          paths:
-            - node_modules
-            - ~/.cache
-      - run:
-          command: npm run test
+      - node/install-npm
+      - node/install-packages
+      - run: npm run test
       - store_artifacts:
           path: 'pact/pacts'
-      - run:
-          command: |
-            npm run build
-            DATE=$(date '+%Y-%m-%d')
-            export BUILD_NUMBER=${DATE}.${CIRCLE_BUILD_NUM}
-            export GIT_REF="$CIRCLE_SHA1"
-            npm run record-build-info
-      - run: # Run linter after build because the integration test code depends on compiled typescript...
```

Comment:
Could we keep this comment in? I don't actually know _what_ it means but seems like somebody thought it was worth pointing out...

Reply:
Done in c45f996: I opted to copy the entire `run lint` step into the integration_test job, following the comment